src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write code here
+const list = document.querySelectorAll('li');
+
+const sortList = (li) => {
```

Comment:
```suggestion
const sortList = () => {
```
Ты этот аргумент используешь?